components/brave_rewards/common/mojom/rewards_page.mojom

```diff
@@ -26,6 +26,30 @@ struct AvailableCountryInfo {
   string default_country_code;
 };
 
+enum AdType {
+  kNewTabPage,
+  kNotification,
+  kSearchResult,
+  kInlineContent
```

Comment:
nit: Could we not use `AdType` in `brave_ads.mojom`?

Reply:
Yes - good call.